pyglmnet/pyglmnet.py

```diff
@@ -498,6 +499,8 @@ def __init__(self, distr='poisson', alpha=0.5,
         self.solver = solver
         self.learning_rate = learning_rate
         self.max_iter = max_iter
+        self.n_iter = 0
```

Comment:
can you also add this parameter to the documentation of the class under a new heading called `Attributes`